packages/plugin-navigation-breadcrumbs/navigation-breadcrumbs.js

```diff
@@ -27,9 +27,8 @@ module.exports = (win = window) => {
       }, true)
 
       // the only way to know about replaceState/pushState is to wrap them… >_<
-
-      if (win.history.replaceState) wrapHistoryFn(client, win.history, 'replaceState', win)
       if (win.history.pushState) wrapHistoryFn(client, win.history, 'pushState', win)
+      if (win.history.replaceState) wrapHistoryFn(client, win.history, 'replaceState', win, true)
```

Comment:
shouldn't this be the other way round i.e. resetting only on `pushState`?

Reply:
thankyou! yes